pkg/git/git_test.go

```diff
@@ -8,14 +8,14 @@ func TestRemoteURL(t *testing.T) {
 	t.Parallel()
 
 	actual := RemoteURL("example.com", "app")
-	expected := "ssh://git@example.com:2222/app.git"
+	expected := "ssh://git@example-builder.com:2222/app.git"
```

Comment:
Yeah I don't think this edge case is worth addressing since we don't route on TLDs. I think it's best to just update the unit tests to `deis.example.com` instead of `example.com` so others don't get tripped up by this either.